speechtotextv1/speech_to_text_v1_adapter.go

```diff
@@ -28,6 +29,23 @@ type RecognizeUsingWebsocketOptions struct {
 	// If true, the service returns interim results as a stream of JSON SpeechRecognitionResults objects.
 	// If false, the service returns a single SpeechRecognitionResults object with final results only.
 	InterimResults *bool `json:"interim_results,omitempty"`
+
+	// If `true`, requests processing metrics about the service's transcription of the input audio. The service returns
+	// processing metrics at the interval specified by the `processing_metrics_interval` parameter. It also returns
+	// processing metrics for transcription events, for example, for final and interim results. By default, the service
+	// returns no processing metrics.
+	ProcessingMetrics *bool `json:"processing_metrics,omitempty"`
+
+	// Specifies the interval in real wall-clock seconds at which the service is to return processing metrics. The
+	// parameter is ignored unless the `processing_metrics` parameter is set to `true`.
+	//
+	// The parameter accepts a minimum value of 0.1 seconds. The level of precision is not restricted, so you can specify
+	// values such as 0.25 and 0.125.
+	//
+	// The service does not impose a maximum value. If you want to receive processing metrics only for transcription events
+	// instead of at periodic intervals, set the value to a large number. If the value is larger than the duration of the
+	// audio, the service returns processing metrics only for transcription events.
+	ProcessingMetricsInterval *float32 `json:"processing_metrics_interval,omitempty"`
```

Comment:
I think we need `audio_metrics` here as well

Reply:
Its enbedded here: https://github.com/watson-developer-cloud/go-sdk/blob/icp4d/speechtotextv1/speech_to_text_v1_adapter.go#L24